docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: JetBrains SFTP
+description: How to use JetBrains IDE to browse remote host with Teleport
+h1: SFTP with JetBrains IDE
+---
+
+This guide explains how to use Teleport and JetBrains IDE to access files with SFTP.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- JetBrains IDE like PyCharm, IntelliJ, Goland etc. [Products](https://www.jetbrains.com/products/#type=ide)
```

Comment:
```suggestion
- JetBrains IDE like PyCharm, IntelliJ, Goland etc. See [Products](https://www.jetbrains.com/products/#type=ide) for a full list of JetBrains IDEs.
```
Otherwise it's not clear what this link points to.